### ISSUE TITLE
Métodos que se quitaron de la librería

### ISSUE THREAD

**Issue by maurodorso** [matbarofex/pyRofex#20]
**Describe the bug**
Que tal, ante todo felicitaciones por el desarrollo! Te quería consultar por unos métodos que dejaron de figurar en la librería pyRofex. Tengo instalada la v.0.3.3 y para homologar mi desarrollo Primary pide la ejecución de ciertos casos de prueba. Algunos de ellos están relacionados a la Risk API (get_positions, get_detailed_position, account_report) y veo que esos métodos dejaron de estar desde la v.0.3.0, ¿Cuál fue el motivo? ¿Ya no se pueden utilizar más o hicieron una librería aparte?

Para probar, busqué los métodos de la v.0.2.0 y los agregué a la v.0.3.3 de pyRofex y recibo respuesta por parte de la API REST: Si me logueo en el entorno LIVE me devuelve que no tengo acceso a mi cuenta y si me logueo en el entorno REMARKET recibo status OK.

**Expected behavior**
Me gustaría saber si se quitaron porque ya no se pueden utilizar, o si es un bug o trasladaron esos métodos a otra librería.

Muchas gracias!

**Comment by fzanuso** [comment]
Hola Mauro, gracias por el comentario. En principio esos metodos no se agregaron en la libreria por una cuestion de tiempos.

Lo que mencionas de que estaban en la version 0.2.0, donde estas viendo que esos metodos estan en esa version? Lo que si hay es un fork del proyecto donde si agregaron esos metodos: https://github.com/wolf-lucas/pyRofex

Tengo pendiente revisar esos cambios que hizo lucas asi los agregamos a la libreria.

**Comment by maurodorso** [comment]
Sí, los métodos que te menciono estaban en la versión 0.2.0 (adjunto la lib).

Buenísimo lo del fork, aunque parece que Lucas le cambió los nombres (get_positions por get_account_position). Por ahora haré una lib aparte para estas funciones hasta que las incorporen en la 0.3.4+ 
Muchas gracias!

[Primary (librería versión vieja con más funciones).zip](https://github.com/matbarofex/pyRofex/files/7023825/Primary.libreria.version.vieja.con.mas.funciones.zip)

**Comment by fzanuso** [comment]
Hola Mauro, se subio una version 0.4.0 con el merge del fork donde se implementaron estos metodos de riesgo.

Cierro Issue.
